packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -299,6 +312,8 @@ async function fetchData({
           __type: type,
         }
       })
+
+      routeResponse = normalizeCollectionResponse(routeResponse)
```

Comment:
Also we should check if that field is defined and is a function we can call, so probably wrapping it with ``if (typeof normalizeCollectionResponse === `Function`)`` would be good idea, because to me this field seems like optional one